### PR TITLE
AQCU-639: handle null labels coming in for callouts.

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

**Comment by mhines-usgs** [comment]
pulling this locally to check out

File: R/utils-limits.R [hunk 1]
```diff
@@ -82,14 +82,14 @@ testCalloutsByView <- function(plot_obj, view_num, xlimits_real, width_char, xra
...
     
     if (!isEmptyOrBlank(callout_args$x)) {  
-      text_len <- nchar(callout_args$labels)
+      text_len <- ifelse(is.na(callout_args$labels),0,nchar(callout_args$labels))
       
       len <- ifelse(is.null(callout_args$length), 0.1, callout_args$length)
       
       xend <- len * xrange * cos(2*pi*(30/360))
       xnew <- callout_args$x + xend + (width_char * text_len) 
+      thing <- xlimits_real[2]
```
Comment (mhines-usgs) [inline R/utils-limits.R]: for debugging?

Comment (kmschoep-usgs) [reply R/utils-limits.R]: oops, yep.

File: R/utils-limits.R [hunk 2]
```diff
@@ -82,8 +82,7 @@ testCalloutsByView <- function(plot_obj, view_num, xlimits_real, width_char, xra
     callout_args <- plot_obj[[view_num]][[i]]
     
     if (!isEmptyOrBlank(callout_args$x)) {  
-      text_len <- nchar(callout_args$labels)
-      
+      text_len <- ifelse(is.na(callout_args$labels),0,nchar(callout_args$labels))
```
Comment (lindsayplatt) [inline R/utils-limits.R:85]: don't need this to check `isEmptyOrBlank(callout_args$labels)` here?

Comment (lindsayplatt) [reply R/utils-limits.R:85]: I guess depends on if the value you are trying to catch is `NA` or `NULL`

Comment (kmschoep-usgs) [reply R/utils-limits.R:85]: We're trying to catch NA, but what I found is that isEmptyOrBlank only returns true if the entire vector is empty or blank.

Comment (lindsayplatt) [reply R/utils-limits.R:85]: Ahhhh I wonder if we should create another method of isEmptyOrBlank for vectors...what else is it not catching that it should?

Comment (mhines-usgs) [reply R/utils-limits.R:85]: Yeah I'm wondering if that's exactly the issue I'm having with another bug right now.. :grimacing:

**Pull request merged**